one_shot_scripts/apply_migrations.js

```diff
@@ -0,0 +1,85 @@
+// Load Environment Variables
+require('dotenv').config();
+
+const mongoose = require('mongoose');
+const carddb = require('../serverjs/cards');
+const Cube = require('../models/cube');
+const Deck = require('../models/deck');
+const Draft = require('../models/draft');
+const GridDraft = require('../models/gridDraft');
+const cubeMigrations = require('../models/migrations/cubeMigrations');
+const deckMigrations = require('../models/migrations/deckMigrations');
+const draftMigrations = require('../models/migrations/draftMigrations');
+const gridDraftMigrations = require('../models/migrations/gridDraftMigrations');
+const { applyPendingMigrationsPre } = require('../models/migrations/migrationMiddleware');
+
+const MIGRATABLE = Object.freeze([
+  { name: 'Cube', model: Cube, migrate: applyPendingMigrationsPre(cubeMigrations) },
+  { name: 'Deck', model: Deck, migrate: applyPendingMigrationsPre(deckMigrations) },
+  { name: 'Draft', model: Draft, migrate: applyPendingMigrationsPre(draftMigrations) },
+  { name: 'GridDraft', model: GridDraft, migrate: applyPendingMigrationsPre(gridDraftMigrations) },
+]);
+
+const migratableDocsQuery = (currentSchemaVersion) => ({
+  $or: [{ schemaVersion: { $exists: false } }, { schemaVersion: { $lt: currentSchemaVersion } }],
```

Comment:
this will considerably slow down the query, and we should probably just do this check after the docs have been loaded

Reply:
Seems fine.